style.ts

```diff
@@ -289,6 +289,11 @@ export interface TextSymbolizer extends BasePointSymbolizer {
    * Unit to use for the haloWidth.
    */
   haloWidthUnit?: DistanceUnit;
+  /**
+   * The opacity of the haloe. A value between 0 and 1.
+   * 0 is none opaque and 1 is full opaque.
```

Comment:
```suggestion
   * 0 means no opacity (transparent) and 1 is fully opaque.
```

I am actually a bit unsure....